templatetags/htk_tags.py

```diff
@@ -219,6 +220,18 @@ def get_request_duration():
     duration = timer.duration()
     return duration
 
+@register.simple_tag(takes_context=True)
+def has_permission(context, codename):
+    request_meta_data = context.get('request', None)
+    request = request_meta_data.get('request', None)
+    user = request.user
+    permission = Permission.objects.get(codename=codename)
+    app_label = permission.content_type.app_label
+    permission_key = '%s.%s' % (app_label, codename)
+    has_permission = user.has_perm(permission_key)
```

Comment:
this feels... circuitous

do they support multiple modes/syntax? like does it just take the permission object, since you already have it? seems pedantic to have to build the key again when you already have the object (and they obviously would have to look up the object again)

Reply:
I am not sure what `obj` refers to. The model obj? the permission obj? In any case the docs say you need to pass in `perm` which is a string such that `app_label.perm_code`